ccd_snr/sections/_model.py

```diff
@@ -0,0 +1,156 @@
+import aastex
+
+__all__ = [
+    "model",
+]
+
+import ccd_snr.figures
+
+
+def model() -> aastex.Section:
+    result = aastex.Section("CCD Model")
+    result.append(
+        r"""
+In this work, we will model the light-sensitive region of the backilluminated 
+\CCD\ sensor as a epitaxial silicon layer with a thickness $D$, which is coated
+with a thin oxide layer of thickness $\delta$ to provide a realistic transmission 
+coefficent.
+The illuminated side of the epitaxial layer is considered to be implanted with ions
+up to a depth $W$ to create the electric field within the sensor."""
+    )
+    subsection_qe = aastex.Subsection("Quantum Efficiency")
+    subsection_qe.append(
+        r"""
+The \QE\ is a common performance metric for measuring \CCD\ sensitivity and is given 
+by \citet{Janesick2001} as
+\begin{equation} \label{quantum-efficiency}
+    \text{QE}(\lambda) = \frac{N_{e}(\lambda)}{N_\gamma}
+                       = T(\lambda) \times \text{IQY}(\lambda) \times \text{CCE}(\lambda),
```

Comment:
Split CCE into two terms, CCE and transmission through non-illuminated side of epitaxial layer